lib/cldr/export/data/base.rb

```diff
@@ -68,26 +63,13 @@ def paths
         private
 
         def merge_paths(paths_to_merge)
-          # Some parts (`ldml`, `ldmlBCP47` amd `supplementalData`) of CLDR data require that you merge all the
-          # files with the same root element before doing lookups.
-          # Ref: https://www.unicode.org/reports/tr35/tr35.html#XML_Format
-          #
-          # The return of this method is a merged XML Nokogiri document.
-          # Note that it technically is no longer compliant with the CLDR `ldml.dtd`, since:
-          # * it has repeated elements
-          # * the <identity> elements no longer refer to the filename
-          #
-          # However, this is not an issue, since #select will find all of the matches from each of the repeated elements,
-          # and the <identity> elements are not important to us / make no sense when combined together.
```

Comment:
Most of this comment has been moved to `DataFile#merge`

The method itself has been rewritten to use `reduce`, which is cleaner IMO.